Makefile

```diff
@@ -467,19 +467,20 @@ out/minikube_$(DEB_VERSION).deb: out/minikube_$(DEB_VERSION)-0_amd64.deb
 	cp $< $@
 
 out/minikube_$(DEB_VERSION)-0_%.deb: out/minikube-linux-%
-	cp -r installers/linux/deb/minikube_deb_template out/minikube_$(DEB_VERSION)
-	chmod 0755 out/minikube_$(DEB_VERSION)/DEBIAN
-	sed -E -i 's/--VERSION--/'$(DEB_VERSION)'/g' out/minikube_$(DEB_VERSION)/DEBIAN/control
-	sed -E -i 's/--ARCH--/'$*'/g' out/minikube_$(DEB_VERSION)/DEBIAN/control
+	PACKAGING_DIRECTORY="out/minikube_$(DEB_VERSION)-$*"
```

Comment:
This is make, and not bash, so this doesn't work - the make variable will not be set further down

You can verify, by looking at the generated commands: `make -n out/minikube_1.16.0.deb`

Reply:
You can use [pattern-specific variables](https://www.gnu.org/software/make/manual/html_node/Pattern_002dspecific.html#Pattern_002dspecific) to declare this as a "local" variable for this target, e.g.:
```
out/minikube_$(DEB_VERSION)-0_%.deb: PACKAGING_DIRECTORY=out/minikube_$(DEB_VERSION)-$*
out/minikube_$(DEB_VERSION)-0_%.deb: out/minikube-linux-%
```
NB: you _must_ put the variable and the pattern rule on separate lines, can't amend the variable as a third item after the pattern in the same line

Reply:
That fixed, now that you said it that obvious, it works now.